pkg/k8s/deployments/crud.go

```diff
@@ -104,6 +107,36 @@ func GetRevisionAnnotatedDeploymentOrFailed(ctx context.Context, dev *model.Dev,
 	return d, nil
 }
 
+func isResourcesRelatedError(errorMessage string) bool {
+	if strings.Contains(errorMessage, "maximum cpu usage") || strings.Contains(errorMessage, "maximum memory usage") {
+		return true
+	}
+	return false
+}
+
+func getResourceLimitError(errorMessage string, dev *model.Dev) error {
+	var errorToReturn string
+	if strings.Contains(errorMessage, "maximum cpu usage") {
+		cpuMaximumRegex, _ := regexp.Compile(`cpu usage per Pod is (\d*\w*)`)
+		maximumCpu := cpuMaximumRegex.FindStringSubmatch(errorMessage)[1]
+		var limitCpuString string
+		if limitCpu, ok := dev.Resources.Limits["cpu"]; ok {
+			limitCpuString = limitCpu.String()
+		}
+		errorToReturn += fmt.Sprintf("Maximum CPU limit per pod is %s, the current value is %s. ", maximumCpu, limitCpuString)
```

Comment:
Let put more focus on the value of okteto manifest that needs to be changed, something like:
```
"The value of resources.limits.cpu in your okteto manifest (%s) exceeds the maximum CPU limit per pod (%s)
```